### PR TITLE
Adjustments in readme

### PR DESCRIPTION
**Pull request by genesisporridge**
Minor adjustments. Removed text duplicates (are found on the website) and added some basic instructions for Windows users.

### CODE REVIEW

File: Taskfile.yml [hunk 1]
```diff
@@ -177,24 +176,28 @@ tasks:
     deps:
       - build:linux
     cmds:
-      - cp scripts/* {{.BUILD_DIR}}/linux-amd64
-      - cp configs/testdata/valkyrie_config.minimal.yml {{.BUILD_DIR}}/linux-amd64/config.yml
+      - "{{.SHELL}} cp scripts/svc.sh {{.BUILD_DIR}}/linux-amd64"
```
Comment (gustavste) [inline Taskfile.yml:179]: cp with * didn't work well in windows

File: Taskfile.yml [hunk 2]
```diff
@@ -121,23 +120,23 @@ tasks:
...
       - ./**/*.go
       - go.mod
+    vars:
+      FILE: '{{if eq .OS "Windows_NT"}}.exe{{end}}'
     generates:
-      - "{{.BUILD_DIR}}/{{.NAME}}"
+      - "{{.BUILD_DIR}}/{{.NAME}}{{.FILE}}"
     cmds:
-      - "{{.BUILD_CMD}} {{.CLI_ARGS}} -o {{.BUILD_DIR}}/{{.NAME}}"
+      - "go build {{.CLI_ARGS}} -o {{.BUILD_DIR}}/{{.NAME}}{{.FILE}}"
```
Comment (gustavste) [inline Taskfile.yml:128]: This task is used by others, and it needs to build including symbols to work `{{.BUILD_CMD}}` includes `-ldflags="-s"` which removes symbols, at least on windows.

**Review by gnirb — APPROVED** [review]
tried all changed tasks, they all still work on macos 👍

File: Taskfile.yml [hunk 3]
```diff
@@ -232,8 +235,9 @@ tasks:
     vars:
       DOCKER_REPO: '{{ default "localhost:5000" .DOCKER_REPO }}'
       TAG: '{{ default "latest" .VERSION }}'
+      SED: '{{if eq .OS "Windows_NT"}}%{$_ -replace "\.git$",""}{{else}}sed "s/\.git$//g" {{end}}' # Remove ".git" from remote url
       SRC_REPO:
-        sh: git ls-remote --get-url | sed 's/\.git$//g'
+        sh: "{{.SHELL}} 'git ls-remote --get-url | {{.SED}}'"
```
Comment (gnirb) [inline Taskfile.yml:240]: Hold on, this one broke for me

Comment (gnirb) [reply Taskfile.yml:240]: Removing the enclosing `'` fixes the issue (but perhaps breaks windows?)

Comment (gustavste) [reply Taskfile.yml:240]: Yeah.. I'll move the whole thing to a Variable then

**Review by gustavste — APPROVED** [review]

**Pull request merged**